### PR TITLE
Add TOML support

### PR DESCRIPTION
**Pull request by kehoecj**
* Added support for TOML file validation
* Replaced filepath.Walk in the fsfinder package with filepath.WalkDir. According to the [go docs](https://pkg.go.dev/path/filepath#Walk) WalkDir is more efficient.
* Replaced log.Println with fmt.Println in the CLI so that the date no longer is displayed in the Help output

### CODE REVIEW

File: pkg/finder/fsfinder.go [hunk 1]
```diff
@@ -21,10 +21,10 @@ func (fsf FileSystemFinder) Find(pathRoot string, fileTypes []filetype.FileType,
 		return nil, err
 	}
 
-	err := filepath.Walk(pathRoot, func(path string, fileInfo fs.FileInfo, err error) error {
+	err := filepath.WalkDir(pathRoot, func(path string, dirEntry fs.DirEntry, err error) error {
 		// determine if directory is in the excludeDirs list
 		for _, dir := range excludeDirs {
-			if fileInfo.IsDir() && fileInfo.Name() == dir {
+			if dirEntry.IsDir() && dirEntry.Name() == dir {
 				//log.Info("Skipping directory")
```
Comment (jd4235) [inline pkg/finder/fsfinder.go:28]:
```suggestion
```

File: pkg/filetype/file_type.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ var JsonFileType = FileType{
 // Instance of the FileType object to
 // represent a YAML file
 var YamlFileType = FileType{
-	"json",
```
Comment (jd4235) [inline pkg/filetype/file_type.go:28]: Interesting. I would assumed the tests would have caught this bug.

File: test/fixtures/good.toml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# This is a TOML document.
```
Comment (jd4235) [inline test/fixtures/good.toml:1]: Should there be a bad.toml to go with this good.toml that will get run by test_cli.go?

**Comment by jackswiney** [comment]
I didn't see anything beyond Jamie's comments. Will approve when those are addressed

**Review by jd4235 — CHANGES_REQUESTED** [review]
See comments.

File: pkg/filetype/file_type.go (hunk 2 above)
Comment (kehoecj) [reply pkg/filetype/file_type.go:28]: We don't use the Name attribute in the FileType struct anywhere for search or validate logic. The purprose was for display/metadata which is why this wasn't caught.

File: test/fixtures/good.toml (hunk 3 above)
Comment (kehoecj) [reply test/fixtures/good.toml:1]: Will add. I may use this opportunity to restructure the fixture directory structure as well. There isn't any organization right now.

Comment (kehoecj) [reply test/fixtures/good.toml:1]: Added a bad.toml file - will save the refactor for later so these changes can get pushed

**Pull request merged**